tools/zbfind

```diff
@@ -930,7 +929,22 @@ class ZBPoller(Thread):
                 self.startinjtime = time.time()
                 self.kb.sniffer_on()
 
-            packetlist = self.kb.pnext()
+            try:
+                packetlist = self.kb.pnext()
+                if packetlist == None:
+                    thresh += 1
+                else:
+                    thresh = 0
+                if thresh > 25:  # thresh helps handle a bug in killerbee firmware for rzusbstick which makes the stick need to be reset.
```

Comment:
25 is kind of arbitrary.  What I noticed was that the RZUSBStick firmware was pooping out after a few context switches of the same reference, and the only way I could reliably figure out that the firmware was in a hung state was by looking at if I had received any packets for a bit.

I tried a few different ways (over a year ago now so hard to remember) but packet-counting was the only feasible, always-reliable method, and it didn't gain a lot of overhead to implement processing-wise.

Ultimately either the firmware issue should be tracked down or we should implement this as a temp fix in the driver handler for the RZUSBStick.